### DISCUSSION TITLE
RuntimeError using scVelo

### DISCUSSION THREAD

**Discussion by Dannyxu123** [theislab/scvelo#897] [Q&A] [answered]
Hello,
     I am currently using scVelo on my laptop(windows system), but when I run the velocity_graph, it always run into bugs, it looks like below:
RuntimeError: 
        An attempt has been made to start a new process before the current process has finished its bootstrapping phase.
     I think it probably has something to do with the fact that windows system could not operate with multiple cores, but when I reduce the n_jobs parameter to none, I get the same bugs. It would be really helpful if someone has any suggestions?

**Comment by WeilerP** [comment, marked as answer]
@Dannyxu123, this looks very similar to #746. Does [this](https://github.com/theislab/scvelo/issues/746#issuecomment-974679256) reply help?

**Reply by Dannyxu123** [reply]
Yes, it seems to work if I add the code below:
 if __name__ == "__main__":
      main()
Thanks a lot!
